tests/integration/snapshots/return_values/sync-metrics_python38_sqs.json

```diff
@@ -1,3 +1,4 @@
+Serverless: Running "serverless" installed locally (in service node_modules)
```

Comment:
What is this? Can we remove it? or mute it in serverless framework somehow?

Reply:
Yeah this one is weird. Looking into it 🤔

Reply:
Had to downgrade my version of serverless to fix this. Seems like the latest versions of the serverless framework will print that log out without some sort of `--quiet` flag.

Reply:
https://github.com/DataDog/datadog-lambda-python/blob/b0158b6f4315a197ed3044f60df35e786cc3bc1b/scripts/run_integration_tests.sh#L144 you can actually remove them this way, but it's ok we can deal with it next time.